src/lib/components/molecules/page-section/style.module.scss

```diff
@@ -0,0 +1,56 @@
+.section {
+    margin-bottom: var(--space-8);
+}
+
+.borderTop::before {
+    display: block;
+    content: "";
+    border-top: 1px solid #dcdcdc;
+    transform: translate(-12px);
+    width: calc(100% + 24px);
+
+    @include mq($from: mobileLandscape) {
+        transform: translate(-20px);
+        width: calc(100% + 40px);
+    }
+
+    @include mq($from: phablet) {
+        transform: none;
+        width: 100%;
+    }
+
+    @include mq($from: leftCol) {
+        transform: translate(-180px);
+        width: calc(100% + 200px);
+    }
+
+    @include mq($from: wide) {
+        transform: translate(-260px);
+        width: calc(100% + 280px);
+    }
+}
+
+.header {
+    padding-top: var(--space-2);
+    margin-bottom: var(--space-5);
+
+    @include mq($from: leftCol) {
+        position: absolute;
+        max-width: 160px;
+        transform: translateX(-170px);
+        margin-bottom: 0;
+    }
+
+    @include mq($from: wide) {
+        max-width: 220px;
+        transform: translateX(-250px);
+    }
+}
+
+.content {
+    padding-top: 0;
+
+    @include mq($from: leftCol) {
+        padding-top: var(--space-2);
```

Comment:
Not quite sure on the CSS involved - but could we align the two pieces of text from the bottom rather than at the top of each piece of text? Obviously if this is how it is on DCR then lets keep it. But looks slightly wrong at desktop. 

<img width="577" alt="Screenshot 2024-03-05 at 12 42 06" src="https://github.com/guardian/interactive-component-library/assets/10324129/68d34568-d450-42c8-988a-f5e00dc30edd">

Reply:
I think that should be the responsibility of the host app. That text there is just sample content, but really could be anything, so I think the default needs to be that whatever is the content is top-aligned with whatever is the header.